cnf-tests/Dockerfile.openshift

```diff
@@ -104,7 +104,7 @@ ENV CNF_TESTS_IMAGE=cnf-tests:${OCP_VERSION}
 ENV DPDK_TESTS_IMAGE=dpdk:${OCP_VERSION}
 
 # python3 is needed for hwlatdetect
-RUN yum install -y lksctp-tools iproute tmux ethtool iputils numactl-libs iptables kmod linuxptp iperf3 python3 nc iptables && \
+RUN yum install -y lksctp-tools iproute tmux ethtool iputils numactl-libs iptables kmod linuxptp iperf3 python3 nc iptables findutils && \
```

Comment:
ok, I missed this but why again do we need this? aren't we moving to konflux as the only pipeline?

Reply:
yes we are but openshift ci uses this dockerfile to generate the u/s images and QE needs ARM supported cnf-tests image.The Arm support via konflux is filed (https://github.com/openshift-kni/cnf-features-deploy/pull/2316) yet need to enhance it, but since there are no plans to support konflux-builds with ARM soon, we want to preserve the openshift ci option and keep relevant files updated.

Reply:
ok, makes sense. Thanks.